include/extractor/guidance/turn_instruction.hpp

```diff
@@ -146,16 +147,18 @@ struct TurnInstruction
     }
 };
 
-static_assert(sizeof(TurnInstruction) == 1, "TurnInstruction does not fit one byte");
+static_assert(sizeof(TurnInstruction) == 3, "TurnInstruction does not fit three byte");
```

Comment:
its currently a typo, but @daniel-j-h might have to modify this anyways. @daniel-j-h can you check this size when you have the switch and make sure to fix the typo when you are on it?